client/src/client-pages/ResultReport/index.tsx

```diff
@@ -104,7 +104,7 @@ const StyledButtons = styled.div`
   right: 0;
 `;
 
-const StyledButtonMargin = styled.div`
+export const StyledButtonMargin = styled.div`
```

Comment:
Uważam, że takie podkradanie styli z totalnie innych komponentów to kiepska praktyka. Zmienimy coś w źródłowym komponencie, nawet nie kapniemy się, że w tym drugim coś się rozjechało.